pkg/policyengine/PolicyDispatcher_test.go

```diff
@@ -97,14 +96,16 @@ func TestIncomingConnectionRequests(t *testing.T) {
 	policy2.To = []policytypes.WorkloadSetOrSelector{{WorkloadSelector: &selectAllSelector}}
 	addPolicy(t, &policy2, ph)
 
-	requestAttr := event.ConnectionRequestAttr{SrcService: svcName, Direction: event.Incoming}
-	connReqResp, err := ph.AuthorizeAndRouteConnection(&requestAttr)
-	require.Equal(t, event.Allow, connReqResp.Action)
+	srcAttrs := policytypes.WorkloadAttrs{policyengine.ServiceNameLabel: svcName}
```

Comment:
how/where are these retrieved in non-test code? We need to implement some way to get attributes beyond just labels being used now.

Also (and unrelated to this specific location/file), while it may be ok to "send" all properties as `map[string]string` on the wire, I wonder if we want to make them explicit by defining the individual attributes on some enclosing struct passed between control plane and the policy engine (ie., explicitly have `Image`, `Labels` etc. when collected before it is exchanged on the wire with the peer)?
It may help in reasoning in the code, about what is needed and being examined for a policy decision, easier.

Reply:
Yes, the current PR is only preparing for the real gathering and transfer of attributes